spec/features/user_can_set_city_location_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'As a user when I visit dashboard' do
+  describe 'I can set city location' do
+    it 'shows me a random restaurant with info' do
+      current_user = User.create!(uid: 1, provider: "google", email: "something@something.com", first_name: "MAC", last_name: "HAG")
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(current_user)
+
+      city = Geocoder.search("Boulder")
```

Comment:
Does this only work for a city or can they also put in a full address?

Reply:
Full Address works as well